tutorials/hyperspectral/RasterStack-RGB-Images-in-R-Using-HSI.Rmd

```diff
@@ -49,11 +47,16 @@ preferably, RStudio loaded on your computer.
 
 ### Data to Download
 
-{% include/dataSubsets/_data_Met-Time-Series.html %}
+{% include/dataSubsets/_data_Imaging-Spec-Data-H5-2020.html %}
+
+*Remember* that the example dataset included here only has 1 out of every 4 bands
```

Comment:
I can't decide if it is worth also repeating this somewhere in the intro for those that skip the grey box.

Reply:
I added another reminder to the "data tip" on line 218. This might also be a good thing to note in the _includes file for this dataset. I think that I _did_ note this in the _includes file in an edit to a different PR, but I can't remember which one at this point, so I will double check once these are merged and published online (writing myself a Trello reminder now).